Gruntfile.js

```diff
@@ -3,12 +3,23 @@ module.exports = function(grunt) {
     pkg: grunt.file.readJSON('package.json'),
     sass: {
       options: {
+        includePaths: [
+          "node_modules/bourbon/app/assets/stylesheets",
+          "node_modules/bourbon-neat/app/assets/stylesheets",
+          "node_modules/font-awesome/scss",
+          "node_modules/normalize.css",
+          "node_modules/neat-omega/core",
+          "node_modules",
+          "scss"
+        ],
         lineNumbers: true,
         sourcemap: 'none'
       },
       dist: {
         files: {
           'css/decanter.css': 'scss/decanter.scss',
+          'css/decanter-no-markup.css': 'scss/decanter-no-markup.scss',
```

Comment:
Two new example files. One that includes decanter without any markup so developers can use all the functions, mixins, or start to add just the parts they want.